### PR TITLE
Arrange coordinates

### PR DESCRIPTION
**Pull request by stellakunzang**
**Fixes issue #:** 

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

**Description:**

Coordinates for tour are now arranged according to longitude or latitude based on the main street where the coordinates are found. 

**What does this solve?:**

The coordinates for the tour were arranged willy-nilly and the routes were needlessly circuitous. 

**Notes:**

There is definitely room for improvement in future iterations! These methods, like the tours, are not as dynamic as we would want them to be in a perfect world. 

- [X] All tests are passing

### CODE REVIEW

File: app/poro/colfax_tour.rb [hunk 1]
```diff
@@ -11,5 +11,6 @@ def colfax_coordinates
     latitudes = StreetArt.where('address ILIKE ?', '%Colfax%').pluck(:latitude)
     longitudes = StreetArt.where('address ILIKE ?', '%Colfax%').pluck(:longitude)
     coordinates = latitudes.zip(longitudes)
+    arrange_coordinates_by_latitude(coordinates)
```
Comment (stellakunzang) [inline app/poro/colfax_tour.rb:14]: I chose whether to use `by_latitude` or `by_longitude` for the tours based on the orientation of the main drag where art is found.

File: app/poro/tour.rb [hunk 2]
```diff
@@ -3,4 +3,16 @@ class Tour
...
+    coordinates.sort do |a, b|
+      a[0] <=> b[0]
+    end
+  end
+
+  def arrange_coordinates_by_longitude(coordinates)
+    coordinates.sort do |a, b|
+      a[1] <=> b[1]
+    end
+  end
```
Comment (stellakunzang) [inline app/poro/tour.rb:17]: These methods are indeed repetitive. They are simple so I didn't think it was worth trying to make one polymorphic method that took a lat or lon argument. If you have any suggestions, I'm happy to adjust!

File: spec/poros/tour_spec.rb [hunk 3]
```diff
@@ -126,4 +126,126 @@
...
+      description: 'Wolf lady mural'
+      })
+
+    tour = RinoTour.new
+
+    expected =
+    [["39.763562", "-104.978627"],
+    ["39.765144", "-104.979897"],
+    ["39.758005", "-104.985765"],
+    ["39.759987", "-104.986479"]]
```
Comment (stellakunzang) [inline spec/poros/tour_spec.rb:186]: I just dropped some coordinates in google maps and rearranged them until it seemed like a logical sequence. Then I wrote the method and tested if it actually arranged the coordinates the way I wanted, and the methods to get the job done were surprisingly simple to write!

**Review by Gallup93 — APPROVED** [review]

File: app/poro/tour.rb (hunk 2 above)
Comment (elguapogordo) [reply app/poro/tour.rb:17]: i've implemented polymorphism for 2-3 methods before. not sure when "worth it" comes into play, i'm still new at this

File: spec/poros/tour_spec.rb (hunk 3 above)
Comment (elguapogordo) [reply spec/poros/tour_spec.rb:186]: nice work!

**Review by elguapogordo — APPROVED** [review]

**Pull request merged**